modules/tinymce/src/plugins/searchreplace/test/ts/module/test/Utils.ts

```diff
@@ -36,6 +39,15 @@ const pOpenDialog = async (editor: Editor) => {
   return await TinyUiActions.pWaitForDialog(editor);
 };
 
+const pOpenDialogWKeyboard = async (editor: Editor) => {
+  if (platform.os.isMacOS()) {
+    TinyContentActions.keystroke(editor, 'F'.charCodeAt(0), { meta: true });
+  } else {
+    TinyContentActions.keystroke(editor, 'F'.charCodeAt(0), { ctrl: true });
+  }
+  return await TinyUiActions.pWaitForDialog(editor);
```

Comment:
The return and await statement combined is unnecessary here, so just use one or the other.
```suggestion
  await TinyUiActions.pWaitForDialog(editor);
```